region/wreckedship/main/Wrecked Ship Energy Tank Room.json

```diff
@@ -687,6 +740,72 @@
       ],
       "flashSuitChecked": true
     },
+    {
+      "link": [1, 2],
+      "name": "Shinespark to Item (X-Mode)",
+      "requires": [
+        {"or": [
+          "Gravity",
+        {"and": [
+          {"not": "f_DefeatedPhantoon"},
+          "canRiskPermanentLossOfAccess"
+        ]}
+        ]},
+        "h_thornXModeShinecharge",
+        {"shinespark": {"frames": 19}}
+      ],
+      "note": [
+        "Arm pump until Samus starts to move again."
+      ],
+      "wallJumpAvoid": true,
+      "flashSuitChecked": true
+    },
+    {
+      "link": [1, 2],
+      "name": "Gain Flash Suit - Spark to Item (X-Mode Spikesuit)",
+      "requires": [
+        "canFlashSuitDoubleSpark",
```

Comment:
we just renamed this tech in #2400 , so it might be good to pull in the recent changes and edit this here before this can get merged. If you need help with that, lmk.